contracts/lib/Errors.sol

```diff
@@ -71,297 +57,454 @@ library Errors {
+    //                             Dispute Module                             //
     ////////////////////////////////////////////////////////////////////////////
 
-    error PILicenseTemplate__ZeroAccessManager();
+    /// @notice Zero address provided for Access Manager in initializer.
+    error DisputeModule__ZeroAccessManager();
 
-    ////////////////////////////////////////////////////////////////////////////
-    //                     LicensorApprovalChecker                            //
-    ////////////////////////////////////////////////////////////////////////////
-    error LicensorApprovalChecker__Unauthorized();
+    /// @notice Zero address provided for License Registry.
+    error DisputeModule__ZeroLicenseRegistry();
 
-    ////////////////////////////////////////////////////////////////////////////
-    //                            Dispute Module                              //
-    ////////////////////////////////////////////////////////////////////////////
+    /// @notice Zero address provided for IP Asset Registry.
+    error DisputeModule__ZeroIPAssetRegistry();
+
+    /// @notice Zero address provided for Access Controller.
+    error DisputeModule__ZeroAccessController();
 
+    /// @notice Zero address provided for Arbitration Policy.
     error DisputeModule__ZeroArbitrationPolicy();
+
+    /// @notice Zero address provided for Arbitration Relayer.
     error DisputeModule__ZeroArbitrationRelayer();
+
+    /// @notice Zero bytes provided for Dispute Tag.
     error DisputeModule__ZeroDisputeTag();
+
+    /// @notice Zero bytes provided for Dispute Evidence.
     error DisputeModule__ZeroLinkToDisputeEvidence();
+
+    /// @notice Not a whitelisted arbitration policy.
     error DisputeModule__NotWhitelistedArbitrationPolicy();
-    error DisputeModule__NotWhitelistedDisputeTag();
+
+    /// @notice Not a whitelisted arbitration relayer.
     error DisputeModule__NotWhitelistedArbitrationRelayer();
+
+    /// @notice Not a whitelisted dispute tag.
+    error DisputeModule__NotWhitelistedDisputeTag();
+
+    /// @notice Not the dispute initiator.
     error DisputeModule__NotDisputeInitiator();
+
+    /// @notice Not in dispute state, the dispute is not IN_DISPUTE.
     error DisputeModule__NotInDisputeState();
+
+    /// @notice Not able to resolve a dispute, either the dispute is IN_DISPUTE or empty.
     error DisputeModule__NotAbleToResolve();
+
+    /// @notice Not a registered IP.
     error DisputeModule__NotRegisteredIpId();
+
+    /// @notice Provided parent IP and the parent dispute's target IP is different.
     error DisputeModule__ParentIpIdMismatch();
+
+    /// @notice Provided parent dispute's target IP is not dispute tagged.
     error DisputeModule__ParentNotTagged();
+
+    /// @notice Provided parent dispute's target IP is not the derivative IP's parent.
     error DisputeModule__NotDerivative();
+
+    /// @notice Provided parent dispute is still in dispute state.
```

Comment:
Provided parent dispute has not been resolved.

Reply:
[9736b8e](https://github.com/storyprotocol/protocol-core-v1/pull/112/commits/9736b8e55e96c43721643e61a88f491c51b900b9)